### PR TITLE
add gradle e2e test

### PR DESCRIPTION
**Pull request by smile37773**

### CODE REVIEW

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -127,5 +127,60 @@ jobs:
...
+        working-directory: ./gradle
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'adopt'
```
Comment (yilims) [inline .github/workflows/package.yml:148]: change distribution to "microsoft"

File: gradle/build.gradle [hunk 2]
```diff
@@ -0,0 +1,28 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.3.0'
+	id 'io.spring.dependency-management' version '1.1.5'
+}
+
+group = 'io.paketo'
```
Comment (yilims) [inline gradle/build.gradle:7]: chang group to "com.example" as the other test case

File: .github/workflows/package.yml (hunk 1 above)
Comment (smile37773) [reply .github/workflows/package.yml]: OK, updated

File: gradle/src/main/java/io/paketo/demo/DemoApplication.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package io.paketo.demo;
```
Comment (yilims) [inline gradle/src/main/java/io/paketo/demo/DemoApplication.java:1]: pls change all the package name to "com.example" as the other test case

Comment (smile37773) [reply gradle/src/main/java/io/paketo/demo/DemoApplication.java]: OK, updated.

File: gradle/build.gradle (hunk 2 above)
Comment (smile37773) [reply gradle/build.gradle]: Updated.

**Review by yilims — APPROVED** [review]

**Pull request merged**